wasp/apps/fliproll.py

```diff
@@ -0,0 +1,118 @@
+ 
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2020 Daniel Thompson
+# Copyright (C) 2021 Evan Rodgers 'setLillie'
+
+"""Digital Coin Flip
+~~~~~~~~~~~~~~~~
+
+Shows a dice roll on screen tap
+
+.. figure:: res/dice_icon.png
```

Comment:
Also, I just noticed this. In your commit there is no `res/dice_icon.png`. Did you mean `res/FlipRollApp.png`?